flexget/utils/imdb.py

```diff
@@ -22,6 +23,7 @@
 
 # this makes most of the titles to be returned in english translation, but not all of them
 requests.headers.update({'Accept-Language': 'en-US,en;q=0.8'})
+requests.headers.update({'X-Forwarded-For': '24.110.%d.%d' % (random.randint(0, 254), random.randint(0,254))})
```

Comment:
I don't know if this should be hardcoded. If IMDB decides to filter out this IP we're fucked. 
Also, some users may prefer to force the language to something different than en.